src/executables/util_cs.c

```diff
@@ -5081,18 +5103,44 @@ flashback (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
-  error = flashback_get_and_show_summary (darray, user, start_time, end_time, &summary_info, &oid_list);
+  error =
+    flashback_get_and_show_summary (darray, user, start_time, end_time, &summary_info, &oid_list, &invalid_class,
+				    &invalid_time);
   if (error != NO_ERROR)
     {
       /* print error message */
       switch (error)
 	{
 	case ER_FLASHBACK_INVALID_TIME:
-	  break;
+	  {
+	    time_t current_time = time (NULL);
```

Comment:
This `current_time` may be different than the time checked on the server side because it can be a gap. Users can be confused with this when `current_time` here printed is bigger than what they input. How about getting the current time at the begin of the utiliy?

Reply:
Yes, that will be better. I will move it to the begin of the utility.